tests/test_mu.py

```diff
@@ -483,7 +484,8 @@ def tearDown(self):
 
     def py_with_pyc(self, name):
         path = os.path.join(self.bench, name)
-        open(path, 'w+').write('42')
+        with open(path, 'w+') as fp:
+            fp.write('42')
```

Comment:
Without this, PyPy doesn't flush the file and it comes up empty in the read in the test below.